zschema/compounds.py

```diff
@@ -11,12 +11,13 @@ def _is_valid_object(name, object_):
 
 class ListOf(Keyable):
 
-    def __init__(self, object_, required=None, max_items=None, doc=None, category=None):
+    def __init__(self, object_, required=None, max_items=None, doc=None, category=None, validator=None):
         self.replace_set("object_", object_)
         self.replace_set("max_items", max_items)
         self.replace_set("category", category)
         self.replace_set("required", required)
         self.replace_set("doc", doc)
+        self.replace_set("validator", validator)
```

Comment:
Does this have a defined default value?

Reply:
https://github.com/zmap/zschema/pull/42/commits/da1e2c4a2d4363bad9df9aded5a1865ffba90104 removes `validator`